android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java

```diff
@@ -300,10 +300,8 @@ private void handleFoundDevice(final ScanResult scanResult, ParcelUuid filteredU
       scanLeDevice(false);
       mUriBeaconConfig = new UriBeaconConfig(getActivity(), new UriBeaconConfigCallback(),
                                              filteredUuid);
-      if (mUriBeaconConfig != null) {
```

Comment:
It's being created one line above, and it's never nulled - am I missing something?